frontend/components/forms/searchable_user_list.jsx

```diff
@@ -12,42 +12,51 @@ class SearchableUserList extends React.Component {
   }
 
   componentDidMount(){
-    if (this.props.modalType === 'EDIT_CHANNEL_MODAL') {
-      this.props.selectUsers(this.props.prevUserIds);
+    const { modalType, selectUsers, prevUserIds } = this.props;
+    if (modalType === 'EDIT_CHANNEL_MODAL') {
+      selectUsers(prevUserIds);
     }
   }
 
   updateSearch(e){
     this.setState({query: e.target.value});
   }
 
-  render(){
+  createMatchEls(){
     const { users,
             selectedUsers,
             currentUserId,
-            selectUser,
-            deselectUser } = this.props;
-
+            selectUser } = this.props;
```

Comment:
same as with the other object destructuring. `users,` on the next line, and the closing curly brace, along with `= this.props`, on the next line